docs/source/tips-and-tricks.rst

```diff
@@ -4,10 +4,77 @@ Tips and Tricks
 .. py:currentmodule:: htmap
 
 
+.. _cli-tips:
+
+Separate job submission/monitoring/collection
+---------------------------------------------
+
+The CLI is useful to monitor and modify ongoing jobs. Generally, in simple use
+cases the HTMap developers write three scripts:
+
+* A script for job submission (which is run once).
+* A script for monitoring jobs (which is run many times).
+* A script to collect results (which is a few times).
+
+Each script uses these commands:
+
+* Submission: HTMap's Python API is primarily used here, possibly through
+  :func:`map`.
+* Monitoring: CLI usage is heavy here, and the command ``htmap status`` is
+  heavily used. If any of the jobs fail, it's easy to diagnose why with the
+  additional commands.
+* Collection: the completed jobs are collected (as mentioned in
+  :ref:`successful-jobs`) and the results are written to disk/etc.
+
+Use of the CLI is useful for debugging when any of the scripts produces an
+error.
```

Comment:
```suggestion
The CLI is useful for debugging when dealing with component holds and execution errors.
It can be used to quickly view the same kind of information as the :class:`Map` API
(though we recommend loading up the map in Python once you need to do anything
more complex than read text).
```